### ISSUE TITLE
"with delay" option to start configuration one by one to be enabled separately

### ISSUE THREAD

**Issue by rkhmelyuk** [rkhmelyuk/multirun#79]
Currently it is always on. To trigger the logic with next configuration has to be start after previous configuration, delay needs to be set to negative value. That is not obvious and bad experience.

Instead, prefer to enable "with delay" option separately.

**Comment by CodingTil** [comment]
I agree this is necessary. I also only found out from this issue that this feature is even possible in the current version by setting the delay to a negative value (thanks btw)
